### ISSUE TITLE
[DataCap Application] アローズコーポレーション

### ISSUE THREAD

**Issue by ClarkYin5566** [filecoin-project/filecoin-plus-large-datasets#101]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: アローズコーポレーション
- Website / Social Media: https://arrows-corp.co.jp/
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):  5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 300TiB
- On-chain address for first allocation: f1qlw5qik62kvrzvpa7bsst65uobtt3jmkrh3ajsq

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
ARROWS was established in July 2006, and its main business is as follows:
・ Housing business (sales / construction / maintenance work such as sales outsourcing, solar power generation system, storage battery system, HEMS, all-electric, EV charger, remodeling, etc.)
・ Material procurement (solar power generation system, storage battery system, HEMS, all-electric, EV charger, etc. wholesale sales)
・ Business development (biomass business, comprehensive sales of housing materials)
```

What is the primary source of funding for this project?
```
Company assets can cover all expenses
```

What other projects/ecosystem stakeholders is this project associated with?
```
No other projects/ecosystem stakeholders
```

## Use-case details

Describe the data being stored onto Filecoin
```
National Solar Radiation Database (NSRDB) is a serially complete collection of hourly and half-hourly values of the three most common measurements of solar radiation – global horizontal, direct normal, and diffuse horizontal irradiance — and meteorological data. These data have been collected at a sufficient number of locations and temporal and spatial scales to accurately represent regional solar radiation climates.
```

Where was the data in this dataset sourced from?
```
https://registry.opendata.aws/nrel-pds-nsrdb/
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://nsrdb.nrel.gov/
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
License: Creative Commons Attribution 3.0 United States License
```

What is the expected retrieval frequency for this data?
```
Anytime
```

For how long do you plan to keep this dataset stored on Filecoin?
```
540 days
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
In Japan or Asia
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Mainly through online transmission, if the storage provider is in Japan, it can be transmitted offline
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
Choose reliable storage providers to ensure that the data can be stored for a long time and can be retrieved at any time
```

How will you be distributing deals across storage providers?
```
Determine the amount of data allocated according to the qualification of the storage provider, such as equipment stability, pledge amount, etc.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Company assets can cover all expenses. If we need help in the future, we will consult you again.
```

**Comment by galen-mcandrew** [comment]
@ClarkYin5566 Can you please clarify your role? Are you a representative of the client Arrows Corp? Is that a storage provider on the Filecoin network? It sounds like the dataset you are referring to is not related to the client.

**Comment by ClarkYin5566** [comment]
> @ClarkYin5566 Can you please clarify your role? Are you a representative of the client Arrows Corp? Is that a storage provider on the Filecoin network? It sounds like the dataset you are referring to is not related to the client.

**Comment by ClarkYin5566** [comment]
@galen-mcandrew I am the representative of the client Arrows Corp. Because one of our main businesses is the sales of solar energy equipment, NSRDB is an important source of data research. We will try to store NSRDB data in the IPFS network, and if feasible, we will provide our own public solar data.

**Comment by ClarkYin5566** [comment]
@galen-mcandrew Hi galen, anything else needs to be provided？

**Comment by galen-mcandrew** [comment]
## Multisig Notary requested
#### Total DataCap requested 
> 5PiB
#### Expected weekly DataCap usage rate 
> 100TiB

**Comment by swatchliu** [comment]
Is there an access portal you developed（not from AWS）we can use to check?

**Comment by ClarkYin5566** [comment]
> Is there an access portal you developed（not from AWS）we can use to check?

@swatchliu Yes please check https://bafybeifrqma42u6dyd7rbvjvtfeux3mtmfmz4so3yimocsig3o6f6gx3ry.ipfs.dweb.link/

**Comment by IreneYoung** [comment]
Hi @ClarkYin5566,  how many storage providers do you intend to make storage deals with?

**Comment by ClarkYin5566** [comment]
@IreneYoung I would choose at least 5 storage providers

**Comment by IreneYoung** [comment]
@ClarkYin5566 That sounds great! Btw, do you need any help on the data onboarding to Filecoin? I can help you with that. Contact me irene@12ships.com.

**Comment by dkkapur** [comment]
@ClarkYin5566 what is your timeline for starting to make deals? Are you in Filecoin Slack?

**Comment by psh0691** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea7yw2ulqkcb26zd7d74kyi3ix7oq2o24aqxkbpuczmtiy5qp7ch4 
#### Address 
> f1qlw5qik62kvrzvpa7bsst65uobtt3jmkrh3ajsq
#### Datacap Allocated
> 50.00TiB
#### Signer Address
> f1qdko4jg25vo35qmyvcrw4ak4fmuu3f5rif2kc7i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea7yw2ulqkcb26zd7d74kyi3ix7oq2o24aqxkbpuczmtiy5qp7ch4

**Comment by Joss-Hua** [comment]
Sounds good. Can you use the corporate email to send an email to filplus@fil.org for confirmation? 
At the same time, need the help of @galen-mcandrew to share the confirmation results with the notaries.

**Comment by psh0691** [comment]
> 
![스크린샷, 2022-08-13 23-17-46](https://user-images.githubusercontent.com/69004801/184498214-a3c740ad-16e4-4648-a148-3576ddd3f530.png)

**Comment by psh0691** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebiay5ib4jfful5okyhxlod4eh4qhxd26tejjwx6nx6kvi7o7m7xy 
#### Address 
> f1qlw5qik62kvrzvpa7bsst65uobtt3jmkrh3ajsq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1qdko4jg25vo35qmyvcrw4ak4fmuu3f5rif2kc7i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebiay5ib4jfful5okyhxlod4eh4qhxd26tejjwx6nx6kvi7o7m7xy
